otter_welcome_buddy/cogs/hiring_timelines.py

```diff
@@ -0,0 +1,42 @@
+import os
+
+from apscheduler.schedulers.asyncio import AsyncIOScheduler
+from discord.ext import commands
+
+from otter_welcome_buddy.common.constants import DAY_ONE_OF_EACH_MONTH_CRON
+from otter_welcome_buddy.common.utils.dates import DateUtils
+from otter_welcome_buddy.formatters import timeline
+
+
+class Timelines(commands.Cog):
+    """Hiring events for every month"""
+
+    def __init__(self, bot, messages_formatter):
+        self.bot = bot
+        self.messages_formatter = messages_formatter
+        self.scheduler: AsyncIOScheduler = AsyncIOScheduler()
+        self.__configure_scheduler()
```

Comment:
Can we modify this in a way that is possible to enable and disable this without the need to change the code?

Reply:
Try to improve your communication by referring the specific change requirement instead of using this. I will put the cron as env variable.

Reply:
I understand that, but I think this request is as simple as:
```python
    @commands.command()
    async def start(self, _ctx):
        self.__configure_scheduler()

    @commands.command()
    async def stop(self, _ctx):
        self.scheduler.stop()
```
Or some modification like a command group or saving the `job_id` to stop just that cronjob, but that can be as a future task. And if there's any problem or we want to pause this until we do a change, we can run a command instead of shutting down the bot or modifying this code.

Reply:
Oh pretty interesting, I wasn't aware that you could to that, I was thinking more of turning off on using some kind of env var.